src/sagemaker/tensorflow/README.rst

```diff
@@ -647,6 +647,39 @@ When the ``deploy`` call finishes, the created SageMaker Endpoint is ready for p
 how to make predictions against the Endpoint, how to use different content-types in your requests, and how to extend the Web server
 functionality.
 
+Deploying directly from model artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you already have existing model artifacts, you can skip training and deploy them directly to an endpoint:
+
+.. code:: python
+
+  from sagemaker.tensorflow import TensorFlowModel
+
+  tf_model = TensorFlowModel(model_data='s3://mybucket/model.tar.gz',
+                             role='MySageMakerRole',
+                             entry_point='entry.py',
+                             name='model_name')
+
+  predictor = tf_model.deploy(initial_instance_count=1, instance_type='ml.c4.xlarge')
+
+You can also optionally specify a pip requirements if you need to install additional packages into the deployed
```

Comment:
s/requirements/requirements file - might also be helpful to link to something like https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format

Reply:
Good idea, done